src/services/WolfDen.Domain/Entity/AttendenceLog.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace WolfDen.Domain.Entity
+{
+    public class AttendenceLog
+    {
+        public int Id { get;private set; }
+        public int EmployeeId { get;private set; }
+        public DateOnly Date { get; private set; }
+        public DateTime Time { get;private set; }
+        public int DeviceId { get;private set; }
+        public string Direction { get;private set; }
+        public virtual Employee Employee { get;private set; }
+        public virtual Device Device { get;private set; }
+
+        public AttendenceLog() { }

```

Comment:
default construcor can be private, these will only be used by EF for  model creation. EF will access the private members as well using reflection